pkg/manager/aws.go

```diff
@@ -143,10 +146,11 @@ func (m *AWS) runComplianceCheck(ctx context.Context) error {
 	if currentCheckoutInfo.EntitledLicenses == requiredLicenses {
 		statusMessage = "Rancher server has the required amount of licenses"
 	} else {
-		statusMessage = fmt.Sprintf("server is not in compliance, wanted %d, but got %d", requiredLicenses, currentCheckoutInfo.EntitledLicenses)
+		statusMessage = fmt.Sprintf("Server is not in compliance with license terms. At least %d more licens(es) are required in %s to become compliant.", requiredLicenses-currentCheckoutInfo.EntitledLicenses, awsCSP)
```

Comment:
AWS should be capitalized. "License terms" is a bit formal, and "server" is too broad. How about: 
```suggestion
		statusMessage = fmt.Sprintf("You have exceeded your licensed node count. At least %d more licens(es) are required in %s to become compliant.", requiredLicenses-currentCheckoutInfo.EntitledLicenses, awsCSP)
```

Reply:
Change made, good suggestion.